src/docs/extend/innerblocks.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Innerblocks
+description: How to use innerblocks in wp-lemon
+---
+
+
+By using innerblocks, you can nest blocks within other blocks. 
+This allows you to create more complex and flexible layouts by placing blocks inside parent blocks.
+
+> **Note:**
+> A single block can only contain one `InnerBlocks` component.
+
+## Building your template
+When using innerblocks it's common to create a template for the innerblocks. 
+This will prefill the InnerBlocks component when inserted and will make it easier to use for you and your clients. 
```

Comment:
```suggestion
This will prefill the `InnerBlocks `component with blocks within the template when inserted and will make it easier to use for you and your clients. 
```